dptb/nn/embedding/__init__.py

```
@@ -1,22 +1,15 @@
     "SE2Descriptor",
-    "Identity",
-    "E3DeePH",
+    "MPNN",
     "Lem",
     "Slem",
     "Trinity",
-    "E3BaseLineModel6",
-    "E3BaseLineModelNonLocal",
-]
\ No newline at end of file
+]
```

Ruff (0.16.3)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)